### PR TITLE
Fix the evolution method of new Pokemon

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: data/pokedex.js [hunk 1]
```diff
@@ -14789,7 +14789,7 @@ let BattlePokedex = {
 		weightkg: 117,
 		color: "White",
 		prevo: "farfetchdgalar",
-		evoType: "levelExtra",
```
Comment (Zarel) [inline data/pokedex.js]: Does it need to level-up at all? If so, `evoCondition` should be reworded with `" after landing"` or something; If not, we need a new `evoType: "other"`.

Comment (KrisXV) [reply data/pokedex.js]: it doesnt and it appears theres other mons that can evolve without leveling too, so ill modify this PR in a bit

Comment (Zarel) [reply data/pokedex.js]: Thanks!

**Comment by KrisXV** [comment]
done @Zarel

**Pull request merged**